worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -25,27 +25,26 @@ To downpatch DS3 for use with Archipelago, use the following instructions from t
-7. Back up your existing game folder in "\Steam\steamapps\common\DARK SOULS III".
-8. Return back to Steam console. Once the download is complete, it should say so along with the temporary local directory in which the depot has been stored. This is usually something like "\Steam\steamapps\content\app_XXXXXX\depot_XXXXXX". Back up this game folder as well.
-9. Delete your existing game folder in "\Steam\steamapps\common\DARK SOULS III", then replace it with your game folder in "\Steam\steamapps\content\app_XXXXXX\depot_XXXXXX".
-10. Back up and delete your save file "DS30000.sl2" in AppData. AppData is hidden by default. To locate it, press Windows Key + R, type %appdata% and hit enter or: open File Explorer > View > Hidden Items and follow "C:\Users\your username\AppData\Roaming\DarkSoulsIII\numbers".
-11. If you did all these steps correctly, you should be able to confirm your game version in the upper left corner after launching Dark Souls III.
+6. Back up your existing game executable `DarkSoulsIII.exe` found in `\Steam\steamapps\common\DARK SOULS III\Game`. This is easily done by renaming the executable to `DarkSoulsIII.exe.bak`.
+7. Return back to Steam console. Once the download is complete, it should say so along with the temporary local directory in which the depot has been stored. This is usually something like `\Steam\steamapps\content\app_XXXXXX\depot_XXXXXX`.
+8. Take the `DarkSoulsIII.exe` from that folder and place it in `\Steam\steamapps\common\DARK SOULS III\Game`.
+9. Back up and delete your save file `DS30000.sl2` in AppData. AppData is hidden by default. To locate it, press Windows Key + R, type `%appdata%` and hit enter or: open File Explorer > View > Hidden Items and follow `C:\Users\<your username>\AppData\Roaming\DarkSoulsIII\numbers`.
+10. If you did all these steps correctly, you should be able to confirm your game version in the upper left corner after launching Dark Souls III.
 
 
 ## Installing the Archipelago mod
 
-Get the dinput8.dll from the [Dark Souls III AP Client](https://github.com/Marechal-L/Dark-Souls-III-Archipelago-client/releases) and 
-add it at the root folder of your game (e.g. "SteamLibrary\steamapps\common\DARK SOULS III\Game")
+Get the `dinput8.dll` from the [Dark Souls III AP Client](https://github.com/Marechal-L/Dark-Souls-III-Archipelago-client/releases) and 
+add it at the root folder of your game (e.g. `SteamLibrary\steamapps\common\DARK SOULS III\Game`)
 
 ## Joining a MultiWorld Game
 
-1. Run Steam in offline mode, both to avoid being banned and to prevent Steam from updating the game files
+1. Run Steam in offline mode, both to avoid being banned
 2. Launch Dark Souls III
-3. Type in "/connect {SERVER_IP}:{SERVER_PORT} {SLOT_NAME}" in the "Windows Command Prompt" that opened
+3. Type in `/connect {SERVER_IP}:{SERVER_PORT} {SLOT_NAME}` in the "Windows Command Prompt" that opened
```

Comment:
Maybe this is out-of-scope for this PR, but I've seen a lot of people struggle with formatting this part. I think an explicit example would help. Example command: `/connect archipelago.gg:38281 "Example Name"`
You could also include the password field in which case you'd want `/connect archipelago.gg:XXXXX "Example Name" password:"Example Password"` and you'd want to change the text:
```suggestion
3. Type in `/connect {SERVER_IP}:{SERVER_PORT} {SLOT_NAME} password:{PASSWORD}` in the "Windows Command Prompt" that opened
```